ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/layout/TutorialLiveShareLayout.kt

```diff
@@ -0,0 +1,153 @@
+@file:Suppress("UNUSED_EXPRESSION")
+
+package org.cru.godtools.tutorial.layout
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Button
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.airbnb.lottie.compose.LottieAnimation
+import com.airbnb.lottie.compose.LottieCompositionSpec
+import com.airbnb.lottie.compose.LottieConstants
+import com.airbnb.lottie.compose.animateLottieCompositionAsState
+import com.airbnb.lottie.compose.rememberLottieComposition
+import org.ccci.gto.android.common.androidx.compose.foundation.text.minLinesHeight
+import org.cru.godtools.base.ui.theme.GodToolsTheme
+import org.cru.godtools.tutorial.Page
+import org.cru.godtools.tutorial.R
+
+@Composable
+internal fun TutorialLiveShareLayout(
+    nextPage: () -> Unit = {},
+    onTutorialAction: (Int) -> Unit = {},
+    page: Page,
+    anim: Int,
+    title: Int,
+    body: Int,
+    body2: Int? = null
+) = GodToolsTheme() {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(
+                top = dimensionResource(R.dimen.tutorial_page_inset_top),
+                bottom = dimensionResource(R.dimen.tutorial_page_inset_bottom)
+            ).verticalScroll(rememberScrollState())
+    ) {
+        Spacer(modifier = Modifier.weight(1f))
+        Box(modifier = Modifier.fillMaxWidth()) {
+            Column() {
+                Spacer(modifier = Modifier.height(290.dp))
+                Spacer(modifier = Modifier.minLinesHeight(1, MaterialTheme.typography.titleLarge))
+                Spacer(modifier = Modifier.minLinesHeight(8, MaterialTheme.typography.bodyMedium))
+            }
```

Comment:
we will need to match this spacing to what the content for this tutorial actually is. Plus because the animation/image is last we will want ot make sure those line up between pages. This means you will probably have to tweak how you do the spacer layout a bit